src/views/DynamicRulesList.vue

```diff
@@ -45,7 +46,7 @@
     </div>
 
     <div class="content no-data-wrapper"
-         v-if="loadingDocCounter || !selectedBranch">
+         v-if="loadingDocCounter || !selectedBranch || !selectedBranch || !globalFiltersData">
```

Comment:
Duplicated `!selectedBranch` check
```suggestion
         v-if="loadingDocCounter || !selectedBranch || !globalFiltersData">
```